### PR TITLE
versioning and changelog and update packages dep

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
## PR Overview
**This PR will address the following Issue/Feature:**

Adding downstream changes to the package dependency range to account for upstream breaking changes in PR https://github.com/fivetran/dbt_twilio_source/pull/8

**This PR will result in the following new package version:**

v0.3.0

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

> _Note_: This release is a 🚨 **breaking change** 🚨 due to breaking changes introduced in the upstream twilio_source package [v0.3.0](https://github.com/fivetran/dbt_twilio_source/releases/tag/v0.3.0) release, where we explicitly cast the below fields as float types and remove any non-numerical characters. 
- `duration, queue_time, num_media, num_segments, count, duration, price, count, usage` 

Please be aware if you were expecting string values from these fields.
### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [ ] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
- [x] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [ ] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

Test with the following deps
- git: https://github.com/fivetran/dbt_twilio_source.git
  revision: bugfix/cast_numeric_values
  warn-unpinned: false

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

**Comment by fivetran-reneeli** [comment]
will regen docs at approval

File: packages.yml
Comment (fivetran-jamie) [inline packages.yml:7]: obligatory reminder to update

**Review by fivetran-jamie — APPROVED** [review]
looks great to me! good to merge after generating docs to capture upstream changes

File: docs/graph_summary.json
Comment (fivetran-avinash) [inline docs/graph_summary.json:1]: @fivetran-reneeli I don't think we bring in the `graph_summary.json`. Can you remove this file from the docs folder.

File: docs/index.html
Comment (fivetran-avinash) [inline docs/index.html:1]: @fivetran-reneeli It appears you deleted the `index.html`. Can you regen docs and add it back in.

**Review by fivetran-avinash — COMMENTED** [review]
@fivetran-reneeli A few notes before approval.

Also, since it's a breaking change, the versions in the README needs to be bumped in [these sections here](https://github.com/fivetran/dbt_twilio/blob/main/README.md#step-2-install-the-package) and [here.](https://github.com/fivetran/dbt_twilio/blob/main/README.md#-does-this-package-have-dependencies)

File: CHANGELOG.md
Comment (fivetran-reneeli) [reply CHANGELOG.md:4]: updated

File: docs/graph_summary.json
Comment (fivetran-reneeli) [reply docs/graph_summary.json]: whoops-- fixed

File: docs/index.html
Comment (fivetran-reneeli) [reply docs/index.html:1]: added back! thanks

**Review by fivetran-avinash — APPROVED** [review]
lgtm

**Pull request merged**
